internal/pkg/deploy/cloudformation/stack/env.go

```diff
@@ -96,14 +98,25 @@ type EnvConfig struct {
 	ForceUpdate         bool
 }
 
+func (cfg *EnvConfig) loadCustomResourceURLs(crs []uploadable) error {
+	if len(cfg.CustomResourcesURLs) != 0 {
+		return nil
+	}
+	bucket, _, err := s3.ParseARN(cfg.ArtifactBucketARN)
+	if err != nil {
+		return fmt.Errorf("parse artifact bucket ARN: %w", err)
+	}
+	cfg.CustomResourcesURLs = customResourceURLs("dummy" /* region is ignored when parsing URLs */, bucket, crs)
```

Comment:
Remind me why we ignore region and does this create invalid urls 🤔

Reply:
👍 OK boss I changed things up so that this question is no longer relevant.

`s3.ParseURL` can now accept both S3 and an HTTPS URI so I don't have to write "dummy". Check out the last commit.